MultiplayerMod/Multiplayer/Configuration/GameEventBindings.cs

```diff
@@ -37,32 +36,39 @@ public class GameEventBindings {
             () => client.Send(new UpdateCursorPosition(client.Player, position))
         );
 
-        BindTools();
+        DragToolEvents.DragComplete += (sender, args) => {
+            // @formatter:off
+            switch (sender) {
```

Comment:
Nit:
As I get not all sender types must be handled here, so it is making impossible to rely on handling all the cases and using default as a catch branch.

Would be great if we can handle all the cases and do not think about what to skip here.

Reply:
> As I get not all sender types must be handled here

Yes, only ones we support, so what is the reason for handling all cases and use default? I mean, we simply don't care about other types and don't need to check for "unusual" values, because we support only these that are explicitly defined, nothing more.

That's why I didn't get
>Would be great if we can handle all the cases and do not think about what to skip here.

All the supported cases are handled, others - don't care, not an error, not an exception, just full ignorance🙂.

Reply:
It is more about writing a code here.

If a new tool will be added our mod will skip it. Which might be fine since it'll be working. But the missing sync part wont noticed.

Reply:
Not sure it's a problem and not sure it can be unnoticed when something isn't synced (especially a tool😂), but you want to make it extra fool-proof? 
Maybe consider this in the future and create an issue?
What do you think about implementation? It could be like an in-game popup effect "This tool isn't supported in multiplayer yet" or it could crash the game with "Sorry, no way this gonna work".

Reply:
Me neither, thats why I said it is `Nit`.
So if you don't see any small change here to make it either ways here - I'm fine with current impl.

Reply:
Okay 🙂
I've created an issue https://github.com/zuev93/oni_multiplayer/issues/59 just not to forget about it.